mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -88,52 +89,51 @@ class MocapFakeGPSPlugin : public plugin::PluginBase
 	void mocap_tf_d_cb(const geometry_msgs::TransformStamped::ConstPtr &trans)
 	{
 		// Throttle incoming messages to 5hz
-		if ((ros::Time::now() - last_pos_time).toSec() < gps_period) 
+		if ((ros::Time::now() - last_pos_time).toSec() < gps_period)
```

Comment:
`ros::Time - ros::Time = ros::Duration`. So remove `.toSec()` and compare two durations.